receipt_dynamo/receipt_dynamo/entities/ocr_job.py

```
@@ -71,6 +73,30 @@ def __post_init__(self) -> None:
         ):
             raise ValueError("receipt_id must be an integer or None")
 
+        if self.reocr_region is not None:
+            if not isinstance(self.reocr_region, dict):
+                raise ValueError("reocr_region must be a dict or None")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)